src/main/kotlin/com/getkeepsafe/dexcount/Tasks.kt

```diff
@@ -215,12 +236,18 @@ abstract class DexMethodCountTaskBase: DefaultTask() {
             val fieldsRemaining = Math.max(MAX_DEX_REFS - tree.fieldCount, 0)
             val classesRemaining = Math.max(MAX_DEX_REFS - tree.classCount, 0)
 
-            out.println("Total methods in $filename: ${tree.methodCount} ($percentMethodsUsed% used)")
-            out.println("Total fields in $filename:  ${tree.fieldCount} ($percentFieldsUsed% used)")
-            out.println("Total classes in $filename:  ${tree.classCount} ($percentClassesUsed% used)")
-            out.println("Methods remaining in $filename: $methodsRemaining")
-            out.println("Fields remaining in $filename:  $fieldsRemaining")
-            out.println("Classes remaining in $filename:  $classesRemaining")
+            if (isAndroidProject) {
+                out.println("Total methods hello in $filename: ${tree.methodCount} ($percentMethodsUsed% used)")
```

Comment:
I think 'hello' is extraneous here?  Integration tests are failing due to it.

Reply:
This probably was a misplaced cursor. I'm glad it's not a password :-)